app/controllers/application_controller.rb

```diff
@@ -11,6 +11,10 @@ def admin_only
     deny_access unless current_user.is_admin?
   end
 
+  def superadmin_or_admin_only
```

Comment:
`User.is_admin?` already checks if the user is an `admin` or `superadmin` https://github.com/eruditionapp/eruditionapp/blob/master/app/models/user.rb#L36

So this authorization is functionally the same as the one above:
```ruby
  def admin_only
    deny_access unless current_user.is_admin?
  end
```

please remove and update all instances of `superadmin_or_admin_only` accordingly :)